src/lib/utils/getHighlightFromSiblings.ts

```diff
@@ -0,0 +1,20 @@
+import unescape from './unescape';
+import { HighlightedParts } from '../../types';
+
+const getHighlightFromSiblings = (parts: HighlightedParts[], i: number) => {
+  const current = parts[i];
+  const isAlphanumeric = new RegExp(/\w/gi);
```

Comment:
this regex can be constructed outside of the function so it doesn't need to be recreated

Reply:
be careful as it's a global regex though, that sometimes has odd behaviour since regex is stateful